suite-native/navigation/src/components/ScreenContent.tsx

```diff
@@ -17,14 +17,23 @@ const screenContentStyle = prepareNativeStyle<{
     insets: EdgeInsets;
     customHorizontalPadding: number;
     customVerticalPadding: number;
-}>((_, { customHorizontalPadding, customVerticalPadding, insets }) => {
-    const { bottom, left, right } = insets;
+    isScrollable: boolean;
+}>((_, { customHorizontalPadding, customVerticalPadding, insets, isScrollable }) => {
+    const { left, right } = insets;
     return {
         flexGrow: 1,
         paddingTop: customVerticalPadding,
-        paddingBottom: Math.max(bottom, customVerticalPadding),
```

Comment:
This might break bottom padding on some devices. I would leave it here and just extend it with scrollable condition as you're doing

Reply:
Where? I only found this: 
```
    extend: {
        condition: !isTabBarVisible,
        style: {
            paddingBottom: Math.max(insets.bottom, customVerticalPadding),
        },
    },
```

Reply:
Yes, this is it. If there is a Tab bar, we can be sure there is no overflow because it handles it. If the tab bar is not present SafeAreInsets are applied.

Reply:
PR description